incubator/appdaemon/0.1.0/test_values.yaml

```diff
@@ -0,0 +1,52 @@
+image:
+  repository: ghcr.io/truecharts/appdaemon
+  pullPolicy: IfNotPresent
+  tag: "v4.0.8"
+
+strategy:
+  type: Recreate
+
+# Configure the Security Context for the Pod
+podSecurityContext:
+  runAsNonRoot: false
+  runAsUser: 0
+  runAsGroup: 0
+  fsGroup: 0
+
+
+env:
+  HA_URL: "http://192.168.5.31:8123"
+  # used to point appdaemon to your home assistant instance
+  TOKEN: "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJjNjNiMzZkM2QwMTg0M2Y4OGQ2Y2JkYjY4MmViNGQ2NiIsImlhdCI6MTYxODk0MDc4OSwiZXhwIjoxOTM0MzAwNzg5fQ.CUXm9K4jSS--qBQZ9DahqFL1YIgIcCK32pfORqiQ0ns"
+  # used to authenticate app daemon to your home assistant instance
+  DASH_URL: "localhost:5050"
```

Comment:
Uhmm... Be aware: When doing tests there is NO Home Assistant instance at all!
Also, you should use the service name, not localhost. Localhost points to the same Container(!) as AppDaemon, not to the Node-host

Reply:
How would you determine the service name during the tests?

Documentation says the service name is $APPNAME.ix-$APPNAME.svc.cluster.local, however during tests, the name seems to be randomly assigned.

Reply:
Like I said: there is no Home Assitant service during the test at all. So I don't understand what you're trying to do here?